src/popup/components/TransferSendForm.vue

```diff
@@ -372,7 +381,12 @@ export default defineComponent({
       token,
     }: Dictionary) {
       if (token) {
-        formModel.value.selectedAsset = availableTokens.value[token] || aeternityCoin.value;
+        if (props.isMultisig && token !== AETERNITY_SYMBOL) {
+          hasMultisigTokenWarning.value = true;
+        } else {
+          formModel.value.selectedAsset = availableTokens.value[token] || aeternityCoin.value;
+          hasMultisigTokenWarning.value = false;
+        }
```

Comment:
I suggest to set the `hasMultisigTokenWarning` as a computed property. It needs to check if the selected asset is not an AE coin and it's a multisig.

Reply:
It can't be done as the token is not set in the form, this is the only place where we receive the info about the scanned token

Reply:
Makes sense, thanks.